lib/active_storage/service/mirror_service.rb

```diff
@@ -18,24 +19,29 @@ def initialize(primary:, mirrors:)
   end
 
   def upload(key, io, checksum: nil)
-    each_service.collect do |service|
-      service.upload key, io.tap(&:rewind), checksum: checksum
+    uploaders = each_service.collect do |service|
+      ActiveStorage::AsyncUploader.new(service, key, checksum: checksum)
     end
+    io.rewind
+    while chunk = io.read(1024)
```

Comment:
This is the chunk size but I don't know where to places the constant. In MirrorService or in the AsyncUploader?